README.adoc

```diff
@@ -58,4 +58,54 @@ can be used to mint a supply-pegged ERC-20: TBTC. For an in-depth explanation
 about the design, see link:docs/rfc/rfc-1.adoc[RFC 1: tBTCv2 Design].
 
 == The Mechanism
-// FIXME
+
+=== Wallet Lifecycle
+
+==== Wallet Creation
+
+We kick off the wallet creation mechanism in
+link:https://github.com/keep-network/keep-core/tree/main/solidity/ecdsa[keep-core/ecdsa]
+via `Wallets.requestNewWallet`, which verifies:
+
+* That we're not currently already creating a wallet
+* If we have an active wallet it either:
+** Is old enough and has over `walletCreationMinBtcBalance` 
+** Has over `walletCreationMaxBtcBalance`
+
+If everything looks good, we kick off the wallet creation mechanism outlined in 
+link:https://github.com/keep-network/keep-core/tree/main/solidity/ecdsa#the-mechanism[Wallet
+Creation] (through keep-core/ecdsa). This leans heavily on the 
+link:https://github.com/keep-network/keep-core/tree/main/solidity/random-beacon#the-mechanism[Random
+Beacon].
+
+Once that is finished, the wallet registry can call
+`Wallets.registerNewWallet`, which sets the new wallet as the active wallet.
```

Comment:
I would rather refer to [`Bridge.__ecdsaWalletCreatedCallback`](https://github.com/keep-network/tbtc-v2/blob/614e0844a8647a7468c383d6994fd789feed5274/solidity/contracts/bridge/Bridge.sol#L909) because that's the function that is called by the `WalletRegistry`.

Reply:
[17a0705](https://github.com/keep-network/tbtc-v2/pull/326/commits/17a0705aec8a2038eff23d6313cedb3a9fd2e106)